spec/grub_cfg_spec.rb

```diff
@@ -25,4 +25,29 @@
       )
     end
   end
+
+  describe "#boot_entries" do
+    it "gets boot entries list with title: and path: elements" do
+      expect(config.boot_entries).to eq(
+        [
+          {
+            title: "openSUSE Leap 42.1",
+            path: "openSUSE Leap 42.1"
+          },
+          {
+            title: "openSUSE Leap 42.1, with Linux 4.1.12-1-default",
+            path: "Advanced options for openSUSE Leap 42.1>openSUSE Leap 42.1, with Linux 4.1.12-1-default"
+          },
+          {
+            title: "openSUSE Leap 42.1, with Linux 4.1.12-1-default (recovery mode)",
+            path: "Advanced options for openSUSE Leap 42.1>openSUSE Leap 42.1, with Linux 4.1.12-1-default (recovery mode)"
+          },
+          {
+            title: "halt",
+            path: "halt"
+          }
+        ]
+      )
+    end
+  end
```

Comment:
here parser is not insteanciated, so it is class methods. In fact it doesn't matter in ruby if object is class or instance of class as long as it have proper methods.

I will add remaining piece to get better coverage